README.md

```diff
@@ -1,2 +1,81 @@
 # node-axosoft
 A node.js client for the Axosoft API
+
+This library is built to support v6 of the Axosoft REST API and has been tested with both axosoft.com hosted accounts and Axosoft installed on premise.  Any questions can be sent to api@axosoft.com.
+
+[Here's](http://developer.axosoft.com/) a link to our full documentation for the Axosoft REST API.
+
+## Installation
+
+  Install with the node package manager [npm](http://npmjs.org):
+
+    $ npm install node-axosoft
```

Comment:
I'd add a `--save` to this. Most package READMEs (or at least the one's I've seen) usually include it.